authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -110,6 +111,28 @@ public ClientRegistrationRepository clientRegistrationRepository(
         OidcClientRegistration.create(securityConfiguration.getAuthentication().getOidc()));
   }
 
+  @Bean
+  @Order(Ordered.HIGHEST_PRECEDENCE)
+  public SecurityFilterChain unprotectedApiAccessSecurityFilterChain(
```

Comment:
Moved this as highest precedence to make sure it applies first regardless of other method based chains added.